src/Resources/ServiceRate.php

```diff
@@ -28,6 +30,11 @@ class ServiceRate implements ServiceRateInterface
     const RELATIONSHIP_CONTRACT = 'contract';
     const RELATIONSHIP_SERVICE_OPTIONS = 'service_options';
 
+    const INCLUDES = [
+        ResourceInterface::TYPE_CONTRACT => self::RELATIONSHIP_CONTRACT,
```

Comment:
Ah so it can be services or contracts.... hmm, can we think of a better name for `included` since `includedServices` wouldnt cover contracts....?

Reply:
I will actually also include the contract, because people will use `$serviceRate->getContract()` as well.